diff --git a/packages/frontend-shared/js/locale.ts b/packages/frontend-shared/js/locale.ts
index 38584fb..2a1e4de 100644
--- a/packages/frontend-shared/js/locale.ts
+++ b/packages/frontend-shared/js/locale.ts
@@ -9,3 +9,3 @@ import type { Locale } from 'i18n';
 // ここはビルド時に const locale = JSON.parse("...") みたいな感じで置き換えられるので top-level await は消える
-export let locale: Locale = await window.fetch(`/assets/locales/${lang}.${version}.json`).then(r => r.json(), () => null);
+export let locale: Locale = await window.fetch(`./locales/${lang}.${version}.json`).then(r => r.json(), () => null);
 
