diff --git a/packages/frontend/src/pages/chat/room.form.vue b/packages/frontend/src/pages/chat/room.form.vue
index 72aeba0..61f5688 100644
--- a/packages/frontend/src/pages/chat/room.form.vue
+++ b/packages/frontend/src/pages/chat/room.form.vue
@@ -53,6 +53,10 @@ const props = defineProps<{
 	room?: Misskey.entities.ChatRoom | null;
 }>();
 
+const emit = defineEmits<{
+	(_: 'messageSent', message: Misskey.entities.ChatMessage): void;
+}>();
+
 const textareaEl = shallowRef<HTMLTextAreaElement>();
 const fileEl = shallowRef<HTMLInputElement>();
 
@@ -198,6 +202,7 @@ function send() {
 			text: text.value ? text.value : undefined,
 			fileId: file.value ? file.value.id : undefined,
 		}).then(message => {
+			emit('messageSent', message);
 			clear();
 		}).catch(err => {
 			console.error(err);
@@ -210,6 +215,7 @@ function send() {
 			text: text.value ? text.value : undefined,
 			fileId: file.value ? file.value.id : undefined,
 		}).then(message => {
+			emit('messageSent', message);
 			clear();
 		}).catch(err => {
 			console.error(err);
