diff --git a/packages/frontend/src/boot/common.ts b/packages/frontend/src/boot/common.ts
index 2b522d3..4133b08 100644
--- a/packages/frontend/src/boot/common.ts
+++ b/packages/frontend/src/boot/common.ts
@@ -90,3 +90,3 @@ export async function common(createVue: () => Promise<App<Element>>) {
 				// fetch with cache: 'no-store' to ensure the latest locale is fetched
-				await window.fetch(`/assets/locales/${lang}.${version}.json`, { cache: 'no-store' }).then(async res => res.status === 200 && await res.text());
+				await window.fetch(`./locales/${lang}.${version}.json`, { cache: 'no-store' }).then(async res => res.status === 200 && await res.text());
 				window.location.reload();
