diff --git a/packages/sw/src/scripts/lang.ts b/packages/sw/src/scripts/lang.ts
index 63ac4ce..7323a28 100644
--- a/packages/sw/src/scripts/lang.ts
+++ b/packages/sw/src/scripts/lang.ts
@@ -35,3 +35,3 @@ class SwLang {
 		// Service Workerは何度も起動しそのたびにlocaleを読み込むので、CacheStorageを使う
-		const localeUrl = `/assets/locales/${await this.lang}.${_VERSION_}.json`;
+		const localeUrl = `/locales/${await this.lang}.${_VERSION_}.json`;
 		let localeRes = await caches.match(localeUrl);
