diff --git a/packages/frontend/build.ts b/packages/frontend/build.ts
--- a/packages/frontend/build.ts
+++ b/packages/frontend/build.ts
@@ -1,9 +1,6 @@
 import * as fs from 'fs/promises';
 import url from 'node:url';
 import path from 'node:path';
 import { execa } from 'execa';
-import locales from 'i18n';
-import { LocaleInliner } from '../frontend-builder/locale-inliner.js'
-import { createLogger } from '../frontend-builder/logger';
 
 // requires node 21 or later
 const __dirname = path.dirname(url.fileURLToPath(import.meta.url));
@@ -20,30 +17,9 @@ async function viteBuild() {
 	});
 }
 
-
-async function buildAllLocale() {
-	const logger = createLogger()
-	const inliner = await LocaleInliner.create({
-		outputDir,
-		logger,
-		scriptsDir: 'scripts',
-		i18nFile: 'src/i18n.ts',
-	})
-
-	await inliner.loadFiles();
-
-	inliner.collectsModifications();
-
-	await inliner.saveAllLocales(locales);
-
-	if (logger.errorCount > 0) {
-		throw new Error(`Build failed with ${logger.errorCount} errors and ${logger.warningCount} warnings.`);
-	}
-}
-
 async function build() {
 	await fs.rm(outputDir, { recursive: true, force: true });
 	await viteBuild();
-	await buildAllLocale();
 }
 
 await build();
