diff --git a/packages/frontend/vite.config.ts b/packages/frontend/vite.config.ts
index 260d121..78932d0 100644
--- a/packages/frontend/vite.config.ts
+++ b/packages/frontend/vite.config.ts
@@ -181,6 +181,3 @@ export function getConfig(): UserConfig {
 			rollupOptions: {
-				input: {
-					i18n: './src/i18n.ts',
-					entry: './src/_boot_.ts',
-				},
+				input: 'index.html',
 				external: externalPackages.map(p => p.match),
@@ -209,3 +206,3 @@ export function getConfig(): UserConfig {
 			cssCodeSplit: true,
-			outDir: __dirname + '/../../built/_frontend_vite_',
+			outDir: __dirname + '/../../webroot',
 			assetsDir: '.',
