diff --git a/packages/frontend/vite.config.ts b/packages/frontend/vite.config.ts
index 260d121..144e0d7 100644
--- a/packages/frontend/vite.config.ts
+++ b/packages/frontend/vite.config.ts
@@ -90,5 +90,5 @@ export function getConfig(): UserConfig {
 
 	return {
-		base: '/vite/',
+		base: '/',
 
 		// The console is shared with backend, so clearing the console will also clear the backend log.
@@ -181,6 +181,6 @@ export function getConfig(): UserConfig {
 			rollupOptions: {
 				input: {
+					main: 'index.html',
 					i18n: './src/i18n.ts',
-					entry: './src/_boot_.ts',
 				},
 				external: externalPackages.map(p => p.match),
