diff --git a/packages/frontend/vite.config.ts b/packages/frontend/vite.config.ts
--- a/packages/frontend/vite.config.ts
+++ b/packages/frontend/vite.config.ts
@@ -90,5 +90,5 @@ export function getConfig(): UserConfig {
 
 	return {
-		base: '/vite/',
+		base: '/',
 
 		// The console is shared with backend, so clearing the console will also clear the backend log.
@@ -172,6 +172,9 @@ export function getConfig(): UserConfig {
 				'safari16',
 			],
 			manifest: 'manifest.json',
+			modulePreload: {
+				polyfill: false,
+			},
 			rolldownOptions: {
 				experimental: {
 					nativeMagicString: true,
@@ -179,5 +182,5 @@ export function getConfig(): UserConfig {
 				input: {
+					main: 'index.html',
 					i18n: './src/i18n.ts',
-					entry: './src/_boot_.ts',
 				},
 				external: externalPackages.map(p => p.match),
